### PR TITLE
bitnami/dataplatform-bp2: Added annotation based discovery in wavefront to the chart

### PR DESCRIPTION
**Pull request by sarvani95**
**Description of the change**
Updated the dataplatform-bp2 chart, updated wavefront with the annotation based discovery feature and made changes to logstash.metrics accordingly in order for wavefront to  fetch metrics from logsash, updated the images of emitter and exporter applications.

**Benefits**
Metrics from Kafka,spark, Elasticsearch and logstash can be auto discovered and fetched by wavefront.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/dataplatform-bp2/Chart.yaml
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/Chart.yaml:4]: update this later with the latest dataplatform-emitter version.

File: bitnami/dataplatform-bp2/Chart.yaml
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/Chart.yaml:60]: Version 9.0.0 is fine

File: bitnami/dataplatform-bp2/values.yaml
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/values.yaml:66]: update the registry/repo/tag once the exporter is released to dockerhub

File: bitnami/dataplatform-bp2/values.yaml
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/values.yaml:452]: update the registry/repo/tag once the emitter is released to dockerhub

File: bitnami/dataplatform-bp2/values.yaml
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/values.yaml:1118]: are the annotations present by default?

File: bitnami/dataplatform-bp2/README.md
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/README.md:396]: Can we have 2 sections here for clarity? 
1. With Tanzu Observability Deployment  
- Mention the auto-discovery default configuration
- seperate heading in case they want to disable auto-discovery

2. Using an existing Tanzu Observability Deployment
- Mention the auto-discovery default configuration
- seperate heading in case they want to disable auto-discovery

File: bitnami/dataplatform-bp2/README.md
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/README.md:410]: Change 'defaulth' to 'default'

File: bitnami/dataplatform-bp2/README.md
Comment (gkulkarni32) [inline bitnami/dataplatform-bp2/README.md:540]: Also add updates the exporter/emitter versions

**Review by gkulkarni32 — COMMENTED** [review]
Thank you Sarvani, added a few comments

File: bitnami/dataplatform-bp2/values.yaml
Comment (Mauraza) [inline bitnami/dataplatform-bp2/values.yaml:1292]: Could you add a space?
```suggestion
      # config:
```

File: bitnami/dataplatform-bp2/values.yaml
Comment (sarvani95) [reply bitnami/dataplatform-bp2/values.yaml:1118]: yes Geeta, there are added in the Bitnami/logstash chart.

**Comment by sarvani95** [comment]
Please do not merge it to master until exporter and emitter images are release to docker hub.

**Comment by Mauraza** [comment]
Hi @sarvani95,

Thanks!. We wait for you to notify us.

**Comment by sarvani95** [comment]
Hey @FraPazGal, updated the emitter image here for BP2 as well.

File: bitnami/dataplatform-bp2/README.md
Comment (FraPazGal) [inline bitnami/dataplatform-bp2/README.md:512]:
```suggestion
```

Comment (FraPazGal) [reply bitnami/dataplatform-bp2/README.md:512]: You can remove this blank line.

File: bitnami/dataplatform-bp2/values.yaml
Comment (FraPazGal) [inline bitnami/dataplatform-bp2/values.yaml:815]:
```suggestion
      ## @param kafka.metrics.jmx.service.annotations [object] Exporter service annotations
      ##
```

**Review by FraPazGal — COMMENTED** [review]
Hello @sarvani95,

Pending the final approval of #8133, I added a couple of comments to fix minor typos. Could you please take a look at them?

**Review by miguelaeh — APPROVED** [review]
LGTM!

@FraPazGal Please run the job for this PR too. Thanks!

**Pull request merged**
